code/__HELPERS/type2type.dm

```diff
@@ -437,49 +437,42 @@ for(var/t in test_times)
 		else
 			. = max(0, min(255, 138.5177312231 * log(temp - 10) - 305.0447927307))
 
+var/global/list/colors_list = list("white" = "#FFFFFF", "black" = "#000000", "gray" = "#808080", "brown" = "#A52A2A", \
+									"red" = "#FF0000", "darkred" = "#8B0000", "crimson" = "#DC143C", "orange" = "#DC143C", \
+									"yellow" = "#FFFF00", "green" = "#008000", "lime" = "#00FF00", "darkgreen" = "#006400", \
+									"cyan" = "#00FFFF", "blue" = "#0000FF", "navy" = "#000080", "teal" = "#008080", \

```

Comment:
they don't, easier to read though because otherwise the line extends into outer space